### ISSUE TITLE
Headlines not sorting by date.

### ISSUE THREAD

**Issue by neilt** [ViennaRSS/vienna-rss#1147]
Vienna is no longer sorting headlines by date or by title. See attached screen grab.  This occurs across all subscriptions and regardless of sort selected in header.

<img width="607" alt="screen shot 2018-05-21 at 8 39 14 am" src="https://user-images.githubusercontent.com/1809796/40310556-c8fa5e10-5cd2-11e8-9891-5811d03fc140.png">

macOS 10.13.4 (17E202)
Vienna Version 3.3.0 :7f06bb62: (6805)

**Comment by DavidMunch** [comment]
Have you set it to sort by the date header? Your screenshot doesn't reveal that information.

**Comment by neilt** [comment]
Yep, I've even switched it back and forth a couple of times and does not clear up.

Here is a better screen grab.

<img width="783" alt="screen shot 2018-05-22 at 5 31 47 am" src="https://user-images.githubusercontent.com/1809796/40357341-7a8445fc-5d81-11e8-83ff-4be04b911f70.png">

**Comment by neilt** [comment]
Never mind, something must have changed, I found the sort in view.  Sorry for the noise.

**Comment by DavidMunch** [comment]
I've noticed that there's some how an interference between the data header, and the read/unread header - Their settings appear to conflict in some cases. Try clicking both of them next time, and see if it helps.

**Comment by neilt** [comment]
The problem was that in 'View -> Sort By' all options were unchecked. This happened because of a recent update (but not IIRC necessarily the last as I've had the problem longer without time to look into it).  When I selected sort by 'Date' everything started working correctly again.

Now that I have selected a 'Sort By', I cannot get back to the state where no sort by option is selected. So this does not appear to be a user error caused by the UI.
